pages/index.js

```diff
@@ -48,7 +50,11 @@ export default function HomePage({ onToggleFavorite, movieInfo }) {
       <Ul>
         {movies?.results?.map((movie) => (
           <Li key={movie.id}>
-            <Link href={"./movies/" + movie.id} key={movie.id}>
+            <Link
+              href={"./movies/" + movie.id}
+              key={movie.id}
+              style={linkStyle}
```

Comment:
please don't use inline styling. You can style Links as follows: `const StyledLink = styled(Link)` and then add the styling as usual for styled components.

Reply:
please use this for every Link you want to style.